apisix/plugins/server-info.lua

```diff
@@ -87,36 +80,35 @@ local function get_boot_time()
 end
 
 
-local function uninitialized_server_info()
+local function uninitialize_server_info()
     local boot_time = get_boot_time()
     return {
         etcd_version     = "unknown",
         hostname         = core.utils.gethostname(),
         id               = core.id.get(),
         version          = core.version.VERSION,
-        up_time          = ngx_time() - boot_time,
         boot_time        = boot_time,
-        last_report_time = -1,
```

Comment:
Note we should also update the manager api or the server-info report cannot be shown on the dashboard.

Reply:
Got it.